tests/MetadataTest.php

```diff
@@ -55,13 +55,13 @@ public function testMetadataFilesReturnArrays()
         $spFiles = $this->getSpMetadataFiles();
         foreach ($spFiles as $file) {
             $returnVal = include $file;
-            $this->assertTrue(is_array($returnVal), 'Metadata file does not return array as expected. File: ' . $file);
+            $this->assertIsArray($returnVal, 'Metadata file does not return array as expected. File: ' . $file);
         }
 
         $idpFiles = $this->getIdPMetadataFiles();
         foreach ($idpFiles as $file) {
             $returnVal = include $file;
-            $this->assertTrue(is_array($returnVal), 'Metadata file does not return array as expected. File: ' . $file);
+            $this->assertIsArray($returnVal, 'Metadata file does not return array as expected. File: ' . $file);
```

Comment:
If you look at the assertions for PHPUnit, you'll notice they are static. Have the same clean up mess in MailAdmin to deal with.

Reply:
> The assertion methods are declared static and can be invoked from any context using PHPUnit\Framework\Assert::assertTrue(), for instance, or using $this->assertTrue() or self::assertTrue(), for instance, in a class that extends PHPUnit\Framework\TestCase. You can even use global function wrappers such as assertTrue().

https://docs.phpunit.de/en/11.2/assertions.html#static-vs-non-static-usage-of-assertion-methods

We use `class AnnouncementTest extends TestCase` , so it is fine as it is.